### ISSUE TITLE
ansible/hacking/env-setup doesn't work to use ansible/hacking/test-module

### ISSUE THREAD

**Issue by david-resnick** [ansible/ansible#11180]
Trying to follow the [Testing Modules](http://docs.ansible.com/developing_modules.html#testing-modules) instructions fails at the `ansible/hacking/test-module -m ./timetest.py` stage.

```
~/proj ❯❯❯ source ansible/hacking/env-setup
running egg_info
creating lib/ansible.egg-info
writing requirements to lib/ansible.egg-info/requires.txt
writing lib/ansible.egg-info/PKG-INFO
writing top-level names to lib/ansible.egg-info/top_level.txt
writing dependency_links to lib/ansible.egg-info/dependency_links.txt
writing manifest file 'lib/ansible.egg-info/SOURCES.txt'
reading manifest file 'lib/ansible.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
no previously-included directories found matching 'v2'
no previously-included directories found matching 'docsite'
no previously-included directories found matching 'ticket_stubs'
no previously-included directories found matching 'packaging'
no previously-included directories found matching 'test'
no previously-included directories found matching 'hacking'
no previously-included directories found matching 'lib/ansible/modules/core/.git'
no previously-included directories found matching 'lib/ansible/modules/extras/.git'
writing manifest file 'lib/ansible.egg-info/SOURCES.txt'

Setting up Ansible to run out of checkout...

PATH=/Users/dresnick/proj/ansible/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/build/apps/bin
PYTHONPATH=/Users/dresnick/proj/ansible/lib:
MANPATH=/Users/dresnick/proj/ansible/docs/man:

Remember, you may wish to specify your host file with -i

Done!

~/proj ❯❯❯ ansible/hacking/test-module -m ./timetest.py
Traceback (most recent call last):
  File "ansible/hacking/test-module", line 38, in <module>
    import ansible.module_common as module_common
ImportError: No module named module_common
```

But sourcing `ansible/v1/hacking/env-setup` works:

```
~/proj ❯❯❯ source ansible/v1/hacking/env-setup
/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python: can't open file 'setup.py': [Errno 2] No such file or directory

Setting up Ansible to run out of checkout...

PATH=/Users/dresnick/proj/ansible/v1/bin:/Users/dresnick/proj/ansible/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/build/apps/bin
PYTHONPATH=/Users/dresnick/proj/ansible/v1:/Users/dresnick/proj/ansible/lib:
MANPATH=/Users/dresnick/proj/ansible/v1/docs/man:/Users/dresnick/proj/ansible/docs/man:

Remember, you may wish to specify your host file with -i

Done!

~/proj ❯❯❯ ansible/hacking/test-module -m ./timetest.py
* including generated source, if any, saving to: /Users/dresnick/.ansible_module_generated
* this may offset any line numbers in tracebacks/debuggers!
***********************************
RAW OUTPUT
{"time": "2015-06-05 14:07:22.072340"}


***********************************
PARSED OUTPUT
{
    "time": "2015-06-05 14:07:22.072340"
}
```

**Comment by bcoca** [comment]
this might need porting to the new v2 engine

**Comment by fgimian** [comment]
Same problem here guys, I ended up checking out the appropriate tag in ansible rather than working with master:

``` bash
cd ansible
git checkout v1.9.2-1
git submodule init
git submodule update
cd ..
```

Perhaps I am missing something, but all ansible dependencies are also needed for this to work, so I just did the following in a virtualenv:

``` bash
pip install ansible/
```

And now everything seems to work correctly :smile: 

Cheers
Fotis

**Comment by bcoca** [comment]
being fixed in #11449 so closing this and keeping track there

**Comment by fgimian** [comment]
Thanks @bcoca :smile:
